src/config/env.ts

```diff
@@ -78,6 +78,7 @@ export interface EnvVars {
 	WEBHOOK_SECRET: string;
 	GITHUB_CLIENT_ID: string;
 	GITHUB_CLIENT_SECRET: string;
+	GITHUB_CLIENT_SECRET_VAULT: string;
```

Comment:
Should we really use a new variable here or just keep the old one?  Would make it simpler in code since we wouldn't have to update it everywhere or have to have conditionals

Reply:
The new variable is just for the feature flag. I'll rename it to `GITHUB_CLIENT_SECRET` once the FF is removed.